AlicerceTests/Extensions/XCTestCase.swift

```diff
@@ -20,8 +20,9 @@ extension XCTestCase {
 
 extension XCTestExpectation {
     func fulfillAfter(_ time: TimeInterval) {
```

Comment:
I can't comment on the `eventually`, but the same applies.

If the goal is to make a helper to wait for some async call, then I think we should rethink this helper to be something like:

```swift
    func expect(withTimeout timeout: TimeInterval = 5, 
                         description: String = "", 
                         closure: @escaping (XCTestExpectation) -> Void) {
        let expectation = expectation(description: description)
        waitForExpectations(timeout: timeout) { _ in
            closure(expectation) // this should call `expectation.fulfill()` when the work is over
        }
    }
```